mvp-0/values-observer.yaml

```diff
@@ -1,6 +1,21 @@
 thanos:
   fullnameOverride: thanos
+  queryFrontend:
+    enabled: true
+    extraFlags:
+    - --query-range.split-interval=12h 
+    - --query-frontend.log-queries-longer-than=10s
+    - --query-frontend.compress-responses
+    - |-
+      --query-range.response-cache-config="config":
+        "max_size": "500MB"
+        "max_size_items": 0
+        "validity": 0s
+      "type": "in-memory"
   query:
+    extraFlags:
+    - --query.auto-downsampling
```

Comment:
Also, what about --query.default-step and `max_source_resolution` URL paramater as mentioned in [thanos tuning docs](https://github.com/SovereignCloudStack/k8s-observability/blob/main/docs/thanos-tuning.md)?

Reply:
Parameter `--query.default-step` has default `1s` as it is mentioned in my doc and if you are using Grafana as your UI, the step is taken from `min_step`.